src/core/src/main/java/org/apache/jmeter/gui/AbstractJMeterGuiComponent.java

```diff
@@ -152,13 +152,12 @@ public String getComment() {
      *
      * @return a NamePanel containing the name of this component
      */
+    @API(status = API.Status.DEPRECATED, since = "5.2.0")
+    @Deprecated
```

Comment:
Could you add JavaDoc explaining the deprecation and what else to use?

Reply:
if only it was Kotlin...